app-new/app_new/__init__.py

```
@@ -0,0 +1,3 @@
+from lib_new import hello
+def main() -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)